includes/class-newspack-popups-model.php

```diff
@@ -222,6 +269,11 @@ public static function generate_popup( $popup ) {
 		$overlay_opacity = absint( $popup['options']['overlay_opacity'] ) / 100;
 		$overlay_color   = $popup['options']['overlay_color'];
 
+		// Add a class to indicate a preview.
+		if ( Newspack_Popups::previewed_popup_id() ) {
+			array_push( $classes, 'newspack-lightbox--preview' );
```

Comment:
With the non-CSS approach to admin bar suppression, is this still needed?

Reply:
Nope, fixed: d1f90f6e827188218c54d6f05ebcca703eaf43a2